openedx/features/course_experience/templates/course_experience/course-updates-fragment.html

```diff
@@ -36,7 +36,22 @@
         </div>
     </header>
     <div class="page-content">
-        ${HTML(updates_html)}
+        % if len(update_list_html) > 0:
```

Comment:
update_list_html is a string in this case - it just represents that the list is now a single html object - I can change the name to make that more clear.